Templates/DefaultProject/Template/ExportScripts/export_standalone_monolithic_engine_centric.py

```diff
@@ -0,0 +1,161 @@
+import pathlib
+import sys
+import time
+
+from o3de.export_project import process_command
+from o3de import manifest
+
+project_json_data = manifest.get_project_json_data(project_path=o3de_context.project_path)
+project_name = project_json_data.get('project_name')
+host_asset_platform = exp.determine_host_asset_platform()
+parser = argparse.ArgumentParser(prog='Exporter for standalone builds',
+                                 description = "Exports a project as standalone to the desired output directory with release layout. "
+                                                "In order to use this script, the engine and project must be setup and registered beforehand. ")
+parser.add_argument('-out', '--output-path', type=pathlib.Path, required=True, help='Path that describes the final resulting Release Directory path location.')
+parser.add_argument('-cfg', '--config', type=str, default='profile', choices=['release', 'profile'],
+                        help='The CMake build configuration to use when building project binaries. If tool binaries are built with this script, they will use profile mode.')
+parser.add_argument('-aof', '--archive-output-format',
+                        type=str,
+                        help="Format of archive to create from the output directory",
+                        choices=["zip", "gzip", "bz2", "xz"], default="zip")
+parser.add_argument('-sl', '--seedlist', type=pathlib.Path, dest='seedlist_paths', action='append',
+                        help='Path to a seed list file for asset bundling. Specify multiple times for each seed list.')
+parser.add_argument('-gf', '--gem-folder-name-to-copy', type=str, dest='gem_folder_names_to_copy', action='append',
+                        help="The name of a Gem folder to include in release. Only specify folder name. This should match what is in the project directory.")
+parser.add_argument('-pfp', '--project-file_pattern-to-copy', type=str, dest='project_file_patterns_to_copy', action='append',
+                        help="Any additional file patterns located in the project directory. File patterns will be relative to the project path.")
+parser.add_argument('-bnmt', '--build-non-mono-tools', action='store_true')
+parser.add_argument('-nmbp', '--non-mono-build-path', type=pathlib.Path, default=None)
+parser.add_argument('-mbp', '--mono-build-path', type=pathlib.Path, default=None)
+parser.add_argument('-nogame', '--no-game-launcher', action='store_true', help='this flag skips building the Game Launcher on a platform if not needed.')
+parser.add_argument('-noserver', '--no-server-launcher', action='store_true', help='this flag skips building the Server Launcher on a platform if not needed.')
+parser.add_argument('-nounified', '--no-unified-launcher', action='store_true', help='this flag skips building the Unified Launcher on a platform if not needed.')
+parser.add_argument('-pl', '--platform', type=str, default=host_asset_platform, choices=['pc', 'linux', 'mac'])
```

Comment:
not currently supported but should eventually include 'ios' and 'android'

Reply:
That will likely be added in a future PR when we're ready to tackle exporting to mobile.

Reply:
Keep in mind we already have a [deployment script for android](https://www.docs.o3de.org/docs/user-guide/platforms/android/ )

Reply:
That may need to be a separate process altogether then. Atm, I believe this export script is best suited to PC games

Reply:
Eventually this script should be part of a `export and deploy` workflow by platform. The android script currently does the packaging and deployment to the device. I think this will be useful for things like 'deploy server launcher to linux' which will uses this script to create the image but then will upload (through ssh or whatever) to a server.